algrtm/src/components/IconsColoredComponent.js

```diff
@@ -0,0 +1,82 @@
+import Image from "next/image";
+import React from "react";
+
+const IconsColoredComponent = () => {
+  return (
+    <div className=" flex justify-between items-stretch mt-[2.75rem] w-full">
+      <Image
+        src="/assets/logos/coloredLogos/python-c-logo.svg"
+        width={64}
+        height={64}
+        alt="Python logo"
+        className="transition-transform duration-300 transform hover:scale-125"
+      ></Image>
+      <Image
+        src="/assets/logos/coloredLogos/pandas-c-logo.svg"
+        width={64}
+        height={64}
+        alt="Pandas logo"
+        className="transition-transform duration-300 transform hover:scale-125"
+      ></Image>
+      <Image
+        src="/assets/logos/hugging-logo.svg"
+        width={64}
+        height={64}
+        alt="Huggling face logo"
+        className="transition-transform duration-300 transform hover:scale-125"
+      ></Image>
+
+      <Image
+        src="/assets/logos/coloredLogos/numpy-c-logo.svg"
+        width={64}
+        height={64}
+        alt="Numpy logo"
+        className="transition-transform duration-300 transform hover:scale-125"
+      ></Image>
+      <Image
+        src="/assets/logos/coloredLogos/redis-c-logo.svg"
+        width={74.488}
+        height={64}
+        alt="Redis logo"
+        className="transition-transform duration-300 transform hover:scale-125"
+      ></Image>
+      <Image
+        src="/assets/logos/coloredLogos/postgreSQL-c-logo.svg"
+        width={64}
+        height={64}
+        alt="PostgreSQL logo"
+        className="transition-transform duration-300 transform hover:scale-125"
+      ></Image>
+      <Image
+        src="/assets/logos/coloredLogos/AWS-c-logo.svg"
+        width={109.215}
+        height={64}
+        alt="AWS logo"
+        className="transition-transform duration-300 transform hover:scale-125"
+      ></Image>
+      <Image
+        src="/assets/logos/coloredLogos/docker-c-logo.svg"
+        width={89.2}
+        height={64}
+        alt="Docker logo"
+        className="transition-transform duration-300 transform hover:scale-125"
+      ></Image>
+      <Image
+        src="/assets/logos/coloredLogos/pyTorch-c-logo.svg"
+        width={64}
+        height={64}
+        alt="PyTorch logo"
+        className="transition-transform duration-300 transform hover:scale-125"
+      ></Image>
+      <Image
+        src="/assets/logos/coloredLogos/streamlit-c-logo.svg"
+        width={117.029}
+        height={64}
+        alt="Streamlit logo"
+        className="transition-transform duration-300 transform hover:scale-125"
+      ></Image>
+    </div>
+  );
+};
+
+export default IconsColoredComponent;
```

Comment:
The website contains two similar sections with icons - one is larger and in color, and the other is smaller and in black and white. We can organize this into one reusable component and through props modify the appearance. The component would look more or less like this:

```jsx
import Image from "next/image";
import React from "react";

const TechIconsComponent = ({ size = "large", color = true }) => {
  const imagesArray = [
    {
      name: "Python",
      large: { width: 64, height: 64 },
      small: { width: 24.1, height: 24 },
    },
    {
      name: "Pandas",
      large: { width: 64, height: 64 },
      small: { width: 24, height: 24 },
    },
    {
      name: "Hugging face",
      large: { width: 64, height: 64 },
      small: { width: 24, height: 24 },
    },
    {
      name: "Numpy",
      large: { width: 64, height: 64 },
      small: { width: 24, height: 24 },
    },
    {
      name: "Redis",
      large: { width: 74.488, height: 64 },
      small: { width: 27.93, height: 24 },
    },
    {
      name: "PostgreSQL",
      large: { width: 64, height: 64 },
      small: { width: 24, height: 24 },
    },
    {
      name: "AWS",
      large: { width: 109.215, height: 64 },
      small: { width: 40.96, height: 24 },
    },
    {
      name: "Docker",
      large: { width: 89.2, height: 64 },
      small: { width: 33.45, height: 24 },
    },
    {
      name: "PyTorch",
      large: { width: 64, height: 64 },
      small: { width: 24, height: 24 },
    },
    {
      name: "Streamlit",
      large: { width: 117.029, height: 64 },
      small: { width: 43.83, height: 24 },
    },
  ];
  return (
    <div className='flex justify-between items-stretch mt-[2.75rem] w-full'>
      {imagesArray.map((image) => {
        console.log(image.name);
        return (
          <Image
            key={image.name}
            src={`/assets/logos/${image.name
              .toLowerCase()
              .split(" ")
              .join("-")}-logo.svg`}
            width={image[size].width}
            height={image[size].height}
            alt={`${image.name.toUpperCase()} logo`}
            className={`transition-transform duration-300 transform hover:scale-125 ${
              !color && "grayscale"
            }`}
          />
        );
      })}
    </div>
  );
};

export default TechIconsComponent;

```

**Required changes:** update the names of all icon files to match the object name and change them to lowercase